lib/models/ObjectMDAzureInfo.ts

```diff
@@ -1,22 +1,24 @@
+export type ObjectMDAzureInfoMetadata = {
```

Comment:
can we type these fields (containerPublicAccess, containerStoredAccessPolicies, containerImmutabilityPolicy, blobType, blobCopyInfo) ?
should we? (not sure they are still used, it may be a remnant of BlobServer...)

Reply:
I think we can but as you said this is not used today in cloudserver. I also see several fields with values I cannot easily type (no information, no usage beyond this class). Thinking from a higher level, I think a debt we have here is to potentially remove useless fields from metadata objects (and potentially don't write fully the fields like `'x-amz-server-side-encryption-customer-algorithm'` but use a dict so we greatly reduce the size of MD, in turn reducing storage, RAM consumption, network usage, impact of oplog listening, etc).

I created OS-932.